Source/Libraries/Adapters/FileAdapters/ProcessLauncher.cs

```diff
@@ -37,6 +29,16 @@
 using GSF.TimeSeries.Adapters;
 using GSF.TimeSeries.Statistics;
 using GSF.Units;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Management;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
```

Comment:
I just want to avoid edits that go back and forth. If you want to switch over to the default behavior in VS, I'll go in and change my settings right now. I'm not sure what our other devs are doing; might need to give everyone a heads-up.

![image](https://github.com/GridProtectionAlliance/gsf/assets/9203145/9a753782-f23d-4320-b3a8-c04ef2229bae)

Reply:
Ah didn't know that option even existed - I'm OK with "unchecking" that

Reply:
Just to be clear we decided on Checking this box?

Reply:
Yes. The decision is pretty arbitrary, but having it checked is consistent with what we've done in the past.